From jarod@redhat.com  Thu Jan 14 17:03:05 2010
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 7 Jan 2010 18:35:58 -0500
Subject: staging: crystalhd: add missing fixes for userspace lib build
To: linux-kernel@vger.kernel.org, Greg KH <greg@kroah.com>
Cc: Scott Davilla <davilla@4pi.com>
Message-ID: <20100107233558.GA23261@redhat.com>


I somehow managed to not actually include these two fixes in the submission
that was committed to the staging tree. libcrystalhd should eventually be
built against the kernel-provided header, and needs the stdint.h include. The
VOID bit is to keep things in sync with the Mac OS X driver and library that
Scott Davilla is also working on.

Signed-off-by: Scott Davilla <davilla@4pi.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/staging/crystalhd/bc_dts_types.h |    6 ++++++
 1 file changed, 6 insertions(+)

--- a/drivers/staging/crystalhd/bc_dts_types.h
+++ b/drivers/staging/crystalhd/bc_dts_types.h
@@ -25,6 +25,10 @@
 #ifndef _BC_DTS_TYPES_H_
 #define _BC_DTS_TYPES_H_
 
+#ifdef __LINUX_USER__  // Don't include these for KERNEL..
+#include <stdint.h>
+#endif
+
 #if defined(_WIN64) || defined(_WIN32)
 typedef uint32_t		U32;
 typedef int32_t			S32;
@@ -62,7 +66,9 @@ typedef int	BOOL;
 typedef uint32_t	ULONG;
 typedef int32_t		LONG;
 typedef void		*HANDLE;
+#ifndef VOID
 typedef void		VOID;
+#endif
 typedef void		*LPVOID;
 typedef uint32_t	DWORD;
 typedef uint32_t	UINT32;
